Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -664,12 +664,10 @@ internal struct TaskCancelEvent {}
 
 internal class TaskHandler<Delegate: HTTPClientResponseDelegate>: RemovableChannelHandler {
     enum State {
-        case idle
-        case bodySent
+        case inactive
+        case sending
```

Comment:
I think we could come up with slightly more prescriptive names? What are wen `sending` here? The request, the response, the head, the body, the trailers? Some of them, all of them?

Same for `sent`

Reply:
done